### PR TITLE
extend deeplinking example

### LINKED ISSUE #2318: Showcase answer_callback_query with deeplinking URL
Instead of using the deeplinking URL directly in text links or inline buttons, one can also use them as `answer_inline_query(url=deeplinking_url)`, as done e.g. by @spoilerobot . I think it would be a good idea to include this option in the `deeplinking.py` example somehow.

### PR DESCRIPTION
**Pull request by starry69**
Closes #2318

### CODE REVIEW

**Comment by starry69** [comment]
That JobQueue fail seems unrelated to me

File: examples/deeplinking.py [hunk 1]
```diff
@@ -22,21 +22,31 @@
...
+    Filters,
+    CallbackContext,
+)
 
 # Enable logging
 from telegram.utils import helpers
 
 logging.basicConfig(
-    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
```
Comment (Bibo-Joshi) [inline examples/deeplinking.py:37]: Why were the quotes changed? did you do that manually or was that black? Same below

**Review by Bibo-Joshi — COMMENTED** [review]
Looks good :) just one question

File: examples/deeplinking.py (hunk 1 above)
Comment (starry69) [reply examples/deeplinking.py:37]: Yep, done by black.

File: examples/deeplinking.py [hunk 2]
```diff
@@ -71,6 +81,23 @@ def deep_linked_level_2(update: Update, context: CallbackContext) -> None:
...
+        reply_markup=InlineKeyboardMarkup(
+            [[InlineKeyboardButton(text="Like this!", callback_data=KEYBOARD_CALLBACKDATA)]]
+        ),
+    )
+
+
+def deep_link_level_3_callback(update: Update, context: CallbackContext) -> None:
+    """Answers CallbackQuery with deeplinking url."""
+    bot = context.bot
+    url = helpers.create_deep_linked_url(bot.get_me().username, USING_KEYBOARD)
```
Comment (Bibo-Joshi) [inline examples/deeplinking.py:95]: doesn't `Bot` have a convenience property for the username? 🤔

File: examples/deeplinking.py (hunk 1 above)
Comment (Bibo-Joshi) [reply examples/deeplinking.py:37]: mh, strange that it didn't change that before … but whatever.

**Review by Bibo-Joshi — COMMENTED** [review]
Found one more thing :)

File: examples/deeplinking.py (hunk 2 above)
Comment (starry69) [reply examples/deeplinking.py:95]: You're right, i just followed how it was done for other deeplinking levels without thinking, will change 👍

**Review by Bibo-Joshi — APPROVED** [review]

**Pull request merged**